src/views/UIOptionsPopup/style.ts

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+
+export const UIOptionsDialog = styled.div`
```

Comment:
I'd add a box shadow to contribute to its floating sense:
`box-shadow: 5px 5px 8px 0px rgba(0,0,0,0.68);`

Reply:
Also, make sure to always check UI elements in responsive mode to make sure they work on mobile. This element looks like this right now:
![Screenshot_2021-02-24_12-12-40](https://user-images.githubusercontent.com/14295292/109053149-a37ec480-7699-11eb-8dba-44b5f67dd651.png)
You should be able to solve this by adding a `max-width: 95%` property. However, you'll have to correct some issues with the ValueSlider overflowing then.

Reply:
Done.  No overflow problem at all!